person.rb

```diff
@@ -1,22 +1,27 @@
 require './nameable'
+require 'securerandom'
 
 class Person < Nameable
+  attr_accessor :name, :age, :rentals
+
   attr_reader :id
-  attr_accessor :name, :age
 
   def initialize(age, name = 'unknown', parent_permission: true)
```

Comment:
- [ ] Hello the parent_permission should have an = instead of :

Reply:
Hi @dasileker please check the last day requirement project 
it's approved before